h5p.classes.php

```diff
@@ -2130,6 +2134,26 @@ public function getDependenciesFiles($dependencies, $prefix = '') {
     return $files;
   }
 
+    public function hasPresave($libraryName)
+    {
+        if( isset($this->h5pD) ){
+            extract(H5PCore::libraryFromString($libraryName));
+            $library = $this->h5pD->getLibrary($machineName, $majorVersion, $minorVersion);
+            if( !is_null($library)){
+                return $this->fs->hasPresave($libraryName, $library['path']);
+            }
+        }
+        return $this->fs->hasPresave($libraryName);
+    }
+
+  public function addPresaveFile(&$assets, $library, $prefix = ''){
+    $this->getDependencyAssets([
```

Comment:
try to keep line length to a max of 80 chars, and split the values up if they're longer.